org.moreunit.plugin/src/org/moreunit/annotation/MoreUnitAnnotationModel.java

```diff
@@ -354,4 +354,12 @@ public void removeAnnotationModelListener(IAnnotationModelListener listener)
     {
         annotationModelListeners.remove(listener);
     }
+
+    private List<MoreUnitAnnotation> copyAnnotations()
+    {
+        synchronized (annotations)
+        {
+            return new ArrayList<MoreUnitAnnotation>(annotations);

```

Comment:
I don't think so, the return value is used in for-each loops and `synchronizedList` does not provide a thread-safe iterator.